### DISCUSSION TITLE
Crashing Due to Physics Error When Physical Contact Occurs

### DISCUSSION THREAD

**Discussion by kojikoshi-m** [isaac-sim/IsaacLab#1474] [Q&A] [answered]
I recently migrated from the OmniIsaacGymEnvs environment with Isaac Sim 2023.1.0 to the latest Isaac Lab with Isaac Sim 4.2. 
While reinforcement learning became operational, the system crashed whenever physical contact occurred. 

This issue did not occur in the OmniIsaacGymEnvs environment.
I was unsure how to resolve this based on the error log alone. 
Could you kindly provide any suggestions or point me in the right direction?
I have attached the log.
[kit_20241128_181847.log](https://github.com/user-attachments/files/17946352/kit_20241128_181847.log)

**Comment by kellyguo11** [comment]
Hi there, unfortunately the error messages in the log are generic GPU errors coming from the physics simulation. It'll be difficult to narrow down the issue without a repro script. In the meantime, you can try increasing the gpu_* buffer dimensions in PhysxCfg and see if that helps - https://isaac-sim.github.io/IsaacLab/main/source/api/lab/omni.isaac.lab.sim.html#omni.isaac.lab.sim.PhysxCfg

**Reply by kojikoshi-m** [reply, marked as answer]
Hi, thank you for your reply.
The problem was solved by changing the collision of a simple cube from Convex Decomposition to Bounding Cube.
